standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -184,7 +184,9 @@ object StandardizationJob {
     }
 
     // If the meta data value sourcecolumn is set override source data column name with the field name
-    val stdRenameSourceColumns = schema.foldLeft(std)((std, field) => renameSourceColumns(std, field, true))
+    val stdRenameSourceColumns = std.select(schema.fields.map{
```

Comment:
This is a minor stylistic thing, but for the sake of uniformity, across the project we follow two things:
* space before opening a brace for a code block (`.map {`rather than `.map{`)
* the input argument with its arrow on the same line as the function accepting the callback:
```
.map { field =>
  doSomething(field)
  doSomethingElse(field)
}
``` 
rather than 
```
.map {
  field => 
    doSomething(field)
    doSomethingElse(field)
}
```
This keeps the code from bloating, because the arrow technically starts a new code block. If you keep the arrow on the same line, you end up flattening the code block.

Reply:
Ok, thanks. Changed here and in the rest